lib/Parser/perrors.h

```diff
@@ -87,3 +87,4 @@ LSC_ERROR_MSG( 1077, ERRDestructNotInit, "Destructuring declarations cannot have
 LSC_ERROR_MSG(1079, ERRInvalidNewTarget, "Invalid use of the 'new.target' keyword")
 LSC_ERROR_MSG(1080, ERRForInNoInitAllowed, "for-in loop head declarations cannot have an initializer")
 LSC_ERROR_MSG(1081, ERRForOfNoInitAllowed, "for-of loop head declarations cannot have an initializer")
+LSC_ERROR_MSG(1082, ERRDefaultParamInStrictMode, "Cannot apply strict mode on functions with non simple parameter list")
```

Comment:
`ERRDefaultParamInStrictMode` should be named something like `ERRNonSimpleParamListStrictMode`